README.md

```diff
@@ -50,5 +55,4 @@ In order to get access to this fork, you need to link your GitHub account to a v
 * Version: alpha (stability and performance improvements pending)
 * GDK repository: [github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
```

Comment:
GDK repository is now listed above so not needed here.

Reply:
I think we might as well have it here

Reply:
It's there for form as you usually have your repo URL on your repo readme so you can find it again when you clone this baby.